### PR TITLE
ttyd: new package

### PR DESCRIPTION
**Pull request by tsl0922**
Maintainer: me
Compile tested: (ramips, PSG1208, LEDE HEAD, r1656)
Run tested: (ramips, PSG1208, LEDE HEAD, r1656)

Description:

Add ttyd package

Signed-off-by: Shuanglei Tao <tsl0922@gmail.com>

### CODE REVIEW

**Comment by diizzyy** [comment]
Please change copyright disclaimer to your own name instead of openwrt.

File: utils/ttyd/Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+define Package/ttyd/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ttyd $(1)/usr/bin/
+
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/ttyd.init $(1)/etc/init.d/ttyd
+endef
+
+$(eval $(call BuildPackage,ttyd))
```
Comment (thess) [inline utils/ttyd/Makefile]: Needs EOL

File: utils/ttyd/files/ttyd.init [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	fi
+	return 0
+}
+
+restart() {
+	logger -t TTYD 'Restarting ttyd service'
+	stop
+	sleep 2 # give time to shutdown
+	start
+}
```
Comment (thess) [inline utils/ttyd/files/ttyd.init]: Needs EOL

**Comment by thess** [comment]
@diizzyy - Are you saying package Makefiles and files should be copyright by the submitter and not the organization(s) involved? That is certainly not what we've been doing all along. Why do you suggest this change?

BTW @tsl0922 - The copyright date needs to be updated.

**Comment by tsl0922** [comment]
@thess Can you give me the full copyright I should use? Thanks.

File: utils/ttyd/Makefile (hunk 1 above)
Comment (tsl0922) [inline utils/ttyd/Makefile]: @thess done

File: utils/ttyd/files/ttyd.init (hunk 2 above)
Comment (tsl0922) [inline utils/ttyd/files/ttyd.init]: done

**Comment by diizzyy** [comment]
@thess 
Unless you're under contract by OpenWRT there's no point in putting anything else than your own.
"An up-to-date copyright notice. Use OpenWrt if no other present or supply your own." This is also the course LEDE follows, https://www.lede-project.org/faq.html (a bit down).

**Comment by thess** [comment]
Seems to me from that statement, either is OK. No need to force a change if author doesn't want to.

@tsl0922 - Since this is a new package either...
> `# Copyright (C) 2016 OpenWrt.org`
or
> `# Copyright (C) 2016 Shuanglei Tao <tsl0922@gmail.com>`

would be fine.

**Comment by diizzyy** [comment]
Preferably would be your own but oh well, I should have phrased myself better.

**Comment by tsl0922** [comment]
Updated to myself since this may makes it more easily to find my contact information from the Makefile, @diizzyy @thess

**Pull request merged**